server/tinylicious/package.json

```diff
@@ -106,7 +106,7 @@
 		"typescript": "~4.5.5"
 	},
 	"engines": {
-		"node": ">=14.0.0"
```

Comment:
This is a breaking change to a published package. If we do this it probably shouldn't be part of this PR.
Also we probably don't need to go all the way to 16.16 for this either.